content/docs/user-guide/building/conda.md

```diff
@@ -0,0 +1,134 @@
+# Conda Environments
+
+Like [virtual environments](/doc/user-guide/building/venv),
+[conda environments](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html)
+follow the same paradigm of isolating dependencies for a package or a model.
+But, they exist globally and are saved in a single location. Further, they need
+not be limited to installation of python packages.
+
+## Description
+
+Currently, Conda based requirements cannot be determined automatically. But, one
+can pass them manually.
+
+In addition to installing conda packages, `pip` based packages (gathered from
+the model) will also be installed in the `conda` environment.
+
+## Preparation
+
+Make sure that `conda` command line utility is installed and is accessible.
+
+### Generating a new conda environment
+
+```cli
+$ mlem build conda --model model --target newenv --conda_reqs.0.package_name xtensor --conda_reqs.1.package_name openssl
```

Comment:
```suggestion
$ mlem build conda --model model --target newenv \
    --conda_reqs.0.package_name xtensor \
    --conda_reqs.1.package_name openssl
```
and in all other places when needed please do the same)

Reply:
done